### PR TITLE
Add android build command

### PR DESCRIPTION
**Pull request by MarcoPolo**
@gabriel

### CODE REVIEW

File: launchd/android.release.plist [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>android.release.test</string>
```
Comment (gabriel) [inline launchd/android.release.plist]: we should use keybase prefix for all these things so... `keybase.android.release.test` or something like that

File: launchd/android.release.plist [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <key>GOPATH</key>
+        <string>/Users/test/go</string>
+        <key>SLACK_TOKEN</key>
+        <string></string>
+        <key>SLACK_CHANNEL</key>
+        <string></string>
+        <key>PATH</key>
+        <string>/sbin:/usr/sbin:/bin:/usr/bin:/usr/local/bin</string>
+        <key>LOG_PATH</key>
+        <string>/Users/test/Library/Logs/android.release.log</string>
```
Comment (gabriel) [inline launchd/android.release.plist]: log should match the label.. here and below

**Comment by gabriel** [comment]
lg2m with comments

File: launchd/android.release.plist (hunk 1 above)
Comment (MarcoPolo) [inline launchd/android.release.plist]: I meant this to just be android.release, and that's how it is on the machine

**Pull request merged**